typescript/cli/examples/submit/strategy/json-rpc-chain-strategy.yaml

```diff
@@ -4,3 +4,9 @@ anvil2:
 anvil3:
   submitter:
     type: jsonRpc
+zksync1:
```

Comment:
Are these local dev chains?

Reply:
Yes, both zksync1 (in-memory node) & zksync2 (docker)